srcpkgs/spacenavd/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'spacenavd'
+pkgname=spacenavd
+version=1.2
+revision=1
+build_style=gnu-configure
+makedepends="libspnav-devel libX11-devel libXtst-devel"
+depends="libspnav"
+short_desc="Userspace driver for 6-dof input devices"
+maintainer="Alexandru Barbur <alex@ctrlc.name>"
+license="GPL-3.0-or-later"
+homepage="http://spacenav.sourceforge.net"
+changelog="https://github.com/FreeSpacenav/spacenavd/releases"
+distfiles="https://github.com/FreeSpacenav/spacenavd/archive/refs/tags/v${version}.tar.gz"
+checksum="4b4096908f842f697a6410a5a873ff4927bd24bd041b6d60b111e90da1178840"
+conf_files="/etc/spnavrc"
+
+post_install() {
+	vinstall ${FILESDIR}/run 755 etc/sv/spacenavd/
```

Comment:
use vsv

Reply:
Fixed in https://github.com/void-linux/void-packages/commit/9cc48924ccc11c7e2ac591643aeaddce28230987